Library/Homebrew/formula_auditor.rb

```diff
@@ -563,6 +563,15 @@ def audit_homepage
 
       return unless DevelopmentTools.curl_handles_most_https_certificates?
 
+      # Skip gnu.org and nongnu.org audit on GitHub runners
+      # See issue: https://github.com/Homebrew/homebrew-core/issues/206757
+      gnu_homepages = [
+        %r{https://www\.gnu\.org/.+},
+        %r{https://www\.nongnu\.org/.+},
+      ]
+
+      return if gnu_homepages.any?(&homepage.method(:match?)) && ENV["HOMEBREW_GITHUB_HOSTED_RUNNER"]
```

Comment:
Probably can combine the two:

```suggestion
      return if homepage.match?(%r{^https?://www\.(?:non)?gnu\.org/.+}) && ENV["HOMEBREW_GITHUB_HOSTED_RUNNER"]
```